spec.bs

```diff
@@ -586,6 +642,8 @@ directly; for authentication the extension can only be accessed via
 :  Operation applicability
 :: [=registration extension|Registration=] and [=authentication extension|authentication=]
 
+Q: Why does this apply to Registration? Should it not just be for authentication?
```

Comment:
As per 'Client extension processing (registration)' below, it needs to relax the ban on creation in a cross-origin iframe.